templates/datadog-agent.yaml

```diff
@@ -0,0 +1,43 @@
+{{ if .Values.datadogAgent.enabled }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: datadog-agent
+  labels:
+    release: datadog-agent
+    istio-injection: disabled
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: datadog-api-key
+  namespace: datadog-agent
+stringData:
+  api-key: {{ .Values.datadogAgent.apiKey }}
+---
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: datadog-agent
+  namespace: kube-system
+spec:
+  bootstrap: true
+  chart: https://github.com/DataDog/helm-charts/releases/download/datadog-3.25.0/datadog-3.25.0.tgz
+  targetNamespace: datadog-agent
+  valuesContent: |
+    datadog:
```

Comment:
Do you only prefer to run this in user nodes? If so, you might need to toleration this taint:

```
https://github.com/acorn-io/hub-system-charts/blob/main/templates/traefik.yaml#L75
```

Reply:
Good call